includes/Stories_Renderer/Renderer.php

```diff
@@ -65,6 +65,18 @@ abstract class Renderer implements RenderingInterface, Iterator {
 	 */
 	const LIGHTBOX_SCRIPT_HANDLE = 'lightbox';
 
+	/**
+	 * Object ID for the Renderer class.
+	 * To enable support for multiple carousels and lightboxes
+	 * on the same page, we needed to identify each Renderer instance.
+	 *
+	 * This variable is used to add appropriate class to the Web Stories
+	 * wrapper.
+	 *
+	 * @var int
+	 */
+	public static $obj_id = 0;
```

Comment:
Why is this static? It is never referenced statically.

Reply:
It also doesn't need to be public. It is only ever outputted to `get_obj_id`.

Reply:
As mentioned in the PR description, we needed to identify different instances of the rendered web stories containers, for that have kept a static property which can be accessed.

You right maybe since we don't need to access it publicly, we don't need it to be public.